internal-log-fetcher/src/trace_consumer.rs

```diff
@@ -44,14 +44,14 @@ impl TraceConsumer {
 
         let stdout_log_file = OpenOptions::new()
             .create(true)
-            .write(true)
+            
             .append(true)
             .open(base_path.join("consumer-stdout.log"))
             .unwrap();
 
         let stderr_log_file = OpenOptions::new()
             .create(true)
-            .write(true)
```

Comment:
Could you comment on the effect of this change?

Reply:
And the `write(true)` call was automatically removed by Cargo Clippy.